### ISSUE TITLE
feeadjuster setting base fees of 1000 msats even though fee-base is set to 0

### ISSUE THREAD

**Issue by dazl** [lightningd/plugins#457]
After updating CLN to v23.05

with no other config changes

fee adjuster now sets base fees of 1000 msats for all channels 

before updating it would adjust ppm fees but leave base fees unchanged at zero.

```
$ lightning-cli listconfigs |grep "\"fee-base\":"
   "fee-base": 0,
```

**Comment by wtogami** [comment]
feeadjuster needs to be itself adjusted for the deprecations in v23.05?

**Comment by chrisguida** [comment]
Fixed by #525
